rb/lib/selenium/webdriver/edge/legacy_support.rb

```diff
@@ -66,17 +64,18 @@ def mouseUp
         end
 
         def mouseMoveTo(element, x = nil, y = nil)
-          params = { :element => element }
+          params = { element: element }
 
           if x && y
-            params.merge! :xoffset => x, :yoffset => y
+            params[:xoffset] = x
+            params[:yoffset] = y
           end
 
           execute :mouseMoveTo, {}, params
         end
 
         def sendKeysToActiveElement(key)
-          execute :sendKeysToActiveElement, {}, :value => key
+          execute :sendKeysToActiveElement, {}, {value: key}
```

Comment:
So, we were being inconsistent in places. The rubocop setting I used for this was to only use curly braces if a previous argument used curly braces as a hash. Additionally, it uses the curly braces for blocks but not for hashes, which seems preferable to me. If it is different elsewhere, it should be made consistent.